release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -28,7 +28,14 @@ module openconfig-bgp-policy {
     It augments the base routing-policy module with BGP-specific
     options for conditions and actions.";
 
-  oc-ext:openconfig-version "8.0.0";
+  oc-ext:openconfig-version "8.2.0";
```

Comment:
Split the change for last-as support to https://github.com/openconfig/public/pull/1221 as discussed offline, which takes version `8.1.0`. This change is expected to merge after that PR, so we use version `8.2.0` after that.